tools/mull-cxx/CLIOptions.cpp

```diff
@@ -139,6 +139,13 @@ opt<bool> tool::EnableAST(
   llvm::cl::desc("Enable \"white\" AST search (disabled by default)"),
   llvm::cl::cat(MullCXXCategory), llvm::cl::init(false));
 
+opt<std::string> tool::GitDiff(
+    "git-diff",
```

Comment:
Would it make sense to call this `experimental-git-diff` for now to indicate alpha/beta testing phase?

Reply:
I don't think so, though I would pick another name, like `git-diff-ref`or something similar.